linera-core/src/local_node.rs

```diff
@@ -90,22 +93,24 @@ where
     S: Storage + Clone + Send + Sync + 'static,
     ViewError: From<S::StoreError>,
 {
+    #[tracing::instrument(level = "trace", skip(self, proposal))]
```

Comment:
Nit: maybe use `skip_all` instead?

```suggestion
    #[tracing::instrument(level = "trace", skip_all)]
```

Reply:
You're right, this instrumentation was generated by `clippy-tracing` :)

Reply:
I've been through these and more carefully considered which fields to record.  Most of them ended up not using `skip_all` (since I added a `Debug` implementation to `ChainClient`, so at the least `self` is useful now).